service/builder/exporters_builder.go

```diff
@@ -193,22 +196,23 @@ func (eb *ExportersBuilder) buildExporter(
 		return nil, fmt.Errorf("exporter factory not found for type: %s", config.Type())
 	}
 
-	exporter := &builtExporter{}
+	exporter := &builtExporter{
+		logger: logger,
+	}
 
 	inputDataTypes := exportersInputDataTypes[config]
 	if inputDataTypes == nil {
 		// TODO  https://github.com/open-telemetry/opentelemetry-collector/issues/294
 		// Move this validation to config/config.go:validateConfig
 		// No data types where requested for this exporter. This can only happen
 		// if there are no pipelines associated with the exporter.
-		eb.logger.Warn("Exporter " + config.Name() +
-			" is not associated with any pipeline and will not export data.")
+		logger.Warn("Exportee is not associated with any pipeline and will not export data.")
```

Comment:
```suggestion
		logger.Warn("Exporter is not associated with any pipeline and will not export data.")
```

Reply:
Did we lose the exporter name?

Reply:
No because is part of the entries that we attached to the log.